src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -172,13 +172,29 @@ public void Listen(int maxIncoming = 20)
             {
                 while (!this.serverCancel.IsCancellationRequested)
                 {
+                    // Used to record any errors occurrring in the thread pool task.
+                    Exception err = null;
```

Comment:
Rename to "error" "or connectionException"

Reply:
Done